src/app/api/comments/likes/route.ts

```diff
@@ -0,0 +1,88 @@
+import prisma from "@/lib/prisma";
+import { NextResponse, type NextRequest } from "next/server";
+
+export async function POST(req: NextRequest) {
+  try {
+    const url = new URL(req.url);
+    const commentId = url.searchParams.get("commentId");
+    const userId = url.searchParams.get("userId");
```

Comment:
I think we might need to swap this for email because the front end does not normally have access to user ids